raiden/tests/integration/network/proxies/test_secret_registry.py

```diff
@@ -46,9 +47,9 @@ def test_register_secret_happy_path(secret_registry_proxy: SecretRegistry, contr
     the SecretRegistered event.
     """
     secret = make_secret()
-    secrethash = sha256(secret).digest()
+    secrethash = SecretHash(sha256(secret).digest())
```

Comment:
You can use the new `sha256_secrethash` https://github.com/raiden-network/raiden/pull/4506/files#diff-a5ac52a95cddc7f20d62a11c55839bb4R6 here